src/components/appointment/translation.tsx

```diff
@@ -9,6 +9,10 @@ const appointmentMessages = {
     finished: { id: 'appointment.*.finished', defaultMessage: '已結束' },
   }),
   AppointmentCard: defineMessages({
+    periodDurationAtMost: {
+      id: 'appointment.text.periodDurationAtMost',
```

Comment:
use the component name as the middle suffix.

'appointment.AppointmentCard.periodDurationAtMost'

Reply:
Eagle-eyed! Adjusted.